testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1987,6 +1987,21 @@ fn test_replay_tooling() {
         .pop()
         .unwrap();
 
+    let (account, _) = client_proxy.get_account_address_from_parameter("0").unwrap();
+    let script_path = workspace_builder::workspace_root()
+        .join("language/tools/transaction-replay/examples/account_exists.move");
+
+    let bisect_result = json_debugger
+        .bisect_transactions_by_script(script_path.to_str().unwrap(), account, 0, txn.version)
+        .unwrap()
+        .unwrap();
+
+    let account_creation_txn = client_proxy
+        .get_committed_txn_by_acc_seq(&["txn_acc_seq", "0000000000000000000000000a550c18", "1", "false"])
+        .unwrap()
+        .unwrap();
+
+    assert_eq!(account_creation_txn.version + 1, bisect_result);
```

Comment:
There is an off-by-one because the bisect tool return you the first version when the script fails, which means the previous version caused the account to be created.